internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceMultiClusterWrapper.java

```diff
@@ -313,4 +317,10 @@ public void removeOneController() {
   public String getRegionName() {
     return regionName;
   }
+
+  public Properties getPubSubClientProperties() {
```

Comment:
Same here: Can we use a Properties object for pubBrokerDetails to avoid recreating it on every call? Alternatively, we could just return an unmodifiable map.